src/include/daos_obj.h

```diff
@@ -287,6 +287,13 @@ typedef struct {
 	uint16_t	kd_csum_len;
 } daos_key_desc_t;
 
+enum daos_io_flags {
+	/* The RPC will be sent to leader replica. */
+	DIOF_TO_LEADER		= 0x1,
+	/* The RPC will be sent to specified replica. */
+	DIOF_TO_SPEC_SHARD	= 0x2,
```

Comment:
these are not part of the public API, they should be moved src/include/daos/object.h

Reply:
OK